<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HuboInteractBalanceManage</class>
 <widget class="QDialog" name="HuboInteractBalanceManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="playBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>start/stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Push</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>256</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPlainTextEdit" name="pushForceXText"/>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pushForceYText"/>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pushForceZText"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
